### PR TITLE
Set Travis badge to /rubygems user

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR regenerates the README (selectively) to have **a current Travis badge**.

## What this does

- Add the badge information to the script which inserts that into the Pandoc document generation
- While regenerating all of the docs, I noted there were two not-up-to-date source document, which are now made up-to-date

## Things I noted

- the README was generated with incorrect markdown relative links (they should be docs/example.md but were to example.md)
- the CodeClimate project needs to be reconfigured for new ownership
- there's no ISSUE_TEMPLATE configured

### CODE REVIEW

**Comment by bronzdoc** [comment]
@olleolleolle you can't push branches to this repo?

**Comment by olleolleolle** [comment]
(For the record, I can not push branches to this repo.)

**Comment by indirect** [comment]
💯 thank you! :raised_hands:

File: docs/gemstash-private-gems.7.md [hunk 1]
```diff
@@ -81,8 +80,7 @@ If your Gemstash server isn’t running, go ahead and start it:
 
     $ gemstash start
 
-Push your test gem using
-    Rubygems:
```
Comment (bronzdoc) [inline docs/gemstash-private-gems.7.md]: I just tried to release a new Gemstash version and noticed our `rake doc` task is generating the docs in this style 🤔

Comment (olleolleolle) [reply docs/gemstash-private-gems.7.md]: IIRC, there were version incompatible details re Pandoc, so I did all I could to have _something_ run.

Perhaps we can make improvements, now.

**Pull request merged**